src/documentation/documentation-hooks.tsx

```diff
@@ -8,7 +8,7 @@ import { fetchToolkit } from "./explore/api";
 import { Toolkit } from "./explore/model";
 import { pullModulesToTop } from "./reference/apidocs-util";
 
-type State =
+export type State =
```

Comment:
We should probably call this something more descriptive.

Reply:
[`2d8e4c6`](https://github.com/microbit-foundation/python-editor-next/pull/452/commits/2d8e4c6fc444cdb8c9a61cc3a6a617baf198eea3)